packages/lib/src/interactions/hooks.ts

```diff
@@ -20,26 +20,22 @@ const ONE_VECTOR = new Vector3(1, 1, 1);
 const MODIFIER_KEYS: ModifierKey[] = ['Alt', 'Control', 'Shift'];
 
 export function useMoveCameraTo() {
-  const { canvasSize, visSize } = useVisCanvasContext();
+  const { visSize, getFovBox } = useVisCanvasContext();
 
   const camera = useThree((state) => state.camera);
   const invalidate = useThree((state) => state.invalidate);
 
   return useCallback(
     (worldPt: Vector3) => {
-      const { position, scale } = camera;
-
+      const { position } = camera;
       const visBox = Box.fromSize(visSize);
-      const fovBox = Box.empty(worldPt)
-        .expandBySize(canvasSize.width * scale.x, canvasSize.height * scale.y)
```

Comment:
This is basically the new implementation of `getFovBox`. The only difference with `getVisibleDomains` is that I need the to create a box for the "wanted" camera FOV, so I have to initialise the FOV box with a custom `center` point. That's the reason for the second, optional `center` parameter, which defaults to `camera.position`.